deploy/tasks/assets.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Deployer;
+
+// parameters
+set('asset_resource_paths', [
+    'src/AppBundle/Resources/es6',
+    'src/AppBundle/Resources/sass'
+]);
+set('asset_build_base', 'web');
```

Comment:
I don't think there's really need for this param, since it is not configurable anyway. It will always be `web` in eZ 2 and `public` in eZ 3.

Reply:
Removed in a78ad9c155293881cc127eeb15d01f8a167af3d2